doc/usermanual/lua/lua_api.xml

```diff
@@ -3551,19 +3551,20 @@ NOTE2: If the parameter is a directory the call is non-blocking; the film object
 </variablelist>
 </section>
 
-<section status="final" id="darktable_database_move_image">
+<section status="draft" id="darktable_database_move_image">
 <title>darktable.database.move_image</title>
 <indexterm>
 <primary>Lua API</primary>
 <secondary>move_image</secondary>
 </indexterm>
 <synopsis>function( 
 	<emphasis><link linkend="darktable_database_move_image_image">image</link></emphasis> : <link linkend="types_dt_lua_image_t">types.dt_lua_image_t</link>, 
-	<emphasis><link linkend="darktable_database_move_image_film">film</link></emphasis> : <link linkend="types_dt_lua_film_t">types.dt_lua_film_t</link>
+	<emphasis><link linkend="darktable_database_move_image_film">film</link></emphasis> : <link linkend="types_dt_lua_film_t">types.dt_lua_film_t</link>,
+	<emphasis><link linkend="darktable_database_move_image_newname">newname</link></emphasis> : string
```

Comment:
enclose this line in brackets, i.e.

`[<emphasis><link linkend="darktable_database_move_image_newname">newname</link></emphasis> : string]`

to denote this argument is optional